packages/eslint-plugin/src/rules/no-unsafe-member-access.ts

```diff
@@ -26,7 +26,7 @@ export default createRule({
     },
     messages: {
       unsafeMemberExpression:
-        'Unsafe member access {{property}} on an `any` value.',
+        'Unsafe member access {{property}} on an {{type}} value.',
```

Comment:
I'd suggest `on value of type {{type}}` (or `on a value of type {{type}}`) here to avoid a/an confusion.

Reply:
🤔 This and the other `no-unsafe-*` rules don't use that phrasing. I think that'd be a separate issue/discussion from this specific PR.